dev-libs/efl/efl-1.20.7-r1.ebuild

```diff
@@ -0,0 +1,309 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit gnome2-utils pax-utils xdg-utils
+
+DESCRIPTION="Enlightenment Foundation Libraries all-in-one package"
+HOMEPAGE="https://www.enlightenment.org/"
+SRC_URI="https://download.enlightenment.org/rel/libs/${PN}/${P}.tar.xz"
+
+LICENSE="BSD-2 GPL-2 LGPL-2.1 ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos ~x64-solaris ~x86-solaris"
+IUSE="avahi +bmp dds connman debug doc drm +eet egl examples fbcon +fontconfig fribidi gif gles glib gnutls gstreamer harfbuzz hyphen +ico ibus ivi jpeg2k libuv luajit neon nls opengl ssl pdf physics pixman postscript +ppm +psd pulseaudio raw scim sdl sound static-libs svg systemd tga tiff tslib unwind v4l valgrind vlc vnc wayland webp X xcf xim xine xpresent xpm"
+
+REQUIRED_USE="
+	?? ( gnutls ssl )
```

Comment:
this has gone too far. the solution before was not wrong, the warning was just bogus, and it is not necessary to make any `REQUIRED_USE` definitions for it. It is similar to the ffmpeg vs. libav situation, just more convoluted. `USE=ssl` means ssl support is entirely optional. Below that, if `gnutls` is disabled, list other options such as `libressl`. So please add back the previous construct in COMMON_DEPEND:
```
	ssl? (
		gnutls? ( >=net-libs/gnutls-3.3.6 )
		!gnutls? (
			!libressl? ( dev-libs/openssl:0= )
			libressl? ( dev-libs/libressl:0= )
		)
	)
```
(minus `libressl` if it really does not work)